src/platform/emotes/emote.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  generateValidator,
+  JSONSchema,
+  ValidateFunction
+} from '../../validation'
+import { Rarity } from '../../dapps/rarity'
+import { I18N } from '../common'
+import { EmoteRepresentation } from './representation'
+import { EmoteData } from './data'
+
+/** @alpha */
+export type Emote = {
+  id: string
+  version: string
+  names: I18N[]
+  descriptions: I18N[]
+  collectionAddress: string
+  rarity: Rarity
+  tags: string[]
+  contents: string[]
```

Comment:
What does the string represent? We usually do ContentMapping. Please change it to an array of content mappings

Reply:
Just using the same convention used by platform team on wearables:
https://github.com/decentraland/common-schemas/blob/10a57e35c94940f6c6a66ef45d102e2099be946a/src/platform/wearables/representation.ts#L13

Reply:
That convention has several implications, to define it, please help me understand: how are we going to serve and deploy emotes?